anki/template/template.py

```diff
@@ -200,20 +205,27 @@ def render_unescaped(self, tag_name=None, context=None):
 
     def clozeText(self, txt, ord, type):
         reg = clozeReg
-        if not re.search(reg%ord, txt):
+        if not re.search(reg % ord, txt):
             return ""
+
         def repl(m):
             # replace chosen cloze with type
+            noSound = removeClozeSound(m.group(3))
```

Comment:
Well, I think the code is better and more readable than where I started.  
Some comments would not hurt.  Maybe it will set an example.  Thanks for poking at it.

It's definitely better than the original.  I'm fighting the temptation to clean up the code unrelated to my changes.